v22.2/sso-sql.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Cluster Single Sign-On (SSO) for Self-hosted CockroachDB
+summary: Overview of Single Sign-On (SSO) for CockroachDB SQL Access, and review of workflows for authenticating human and bot users, and for configuring the feature.
+toc: true
+docs_area: manage
+---
+
+## What is Cluster Single Sign-On (SSO)?
+
+Cluster SSO allows users to access the SQL interface of a CockroachDB cluster using a JWT auth token issued by a customer-managed identity provider (IdP).
+
+This page discusses use cases for authenticating to {{ site.data.products.core }} clusters. You might instead be looking for [Cluster Single Sign-On (SSO) for Cockroach Cloud](../cockroachcloud/cloud-sso-sql.html) or [Single Sign-On (SSO) for CockroachDB Cloud organizations](../cockroachcloud/cloud-org-sso.html).
+
+**Prerequisites**
+
+- **IdP:**
+
+	You must have the ability to create identities and issue JSON Web Token (JWT) formatted auth tokens.
+
+	This [Cockroach Labs blog post](https://www.cockroachlabs.com/blog/) covers and provides further resources for a variety of token-issuing use cases, including using Okta and Google Cloud Platform to issue tokens.
```

Comment:
The link to the blog post is not going to a specific blog post, it seems like it should?

Reply:
It hasn't been posted yet, I'll update it when that blog post is available.